src/modules/Schedule/components/Schedule/components/ClassCard.tsx

```diff
@@ -56,7 +70,7 @@ export function ClassCard({ scheduleClass }: ClassCardProps) {
         {isWide && courseNo} {abbrName}
       </ClassCardTypography>
       <ClassCardTypography variant="caption">
-        {teachers.join(', ')} | {building} {room}
+        Sec {sectionNo} - {teachers.join(', ')}
```

Comment:
Should document this change on this Github PR detail?

Reply:
oh yeah, thanks for mentions